scripts/nginx/authelia.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+#
+# Get our externial IP
+ex_ip="$(ip -br a | sed -n 2p | awk '{ print $3 }' | cut -f1 -d'/')"
+# Create an /etc/nginx/apps subdirectory we need
+mkdir -p "/etc/nginx/apps/authelia"
+# Fix nginx to not try to load a directory when using an include
+sed 's|include /etc/nginx/apps/\*;|include /etc/nginx/apps/*.conf;|g' -i /etc/nginx/sites-enabled/default
+# Create the main reverse proxy - it contains 2 location driectoive for this example.
+cat > "/etc/nginx/apps/authelia.conf" << AUTHELIA_NGINX_1
+set \$upstream_authelia http://127.0.0.1:9091;
```

Comment:
9091 is already the default RPC port for transmission FYI. Every nth user users has port 9091+n-1, so you might want to go under 9091 maybe.